src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -16,6 +17,9 @@ import { openAuthToFollowSave } from "v2/Utils/openAuthModal"
 import { ArtistSeriesHeader_artistSeries } from "v2/__generated__/ArtistSeriesHeader_artistSeries.graphql"
 import { useSystemContext } from "v2/Artsy"
 import { Intent } from "@artsy/cohesion"
+import { resize } from "v2/Utils/resizer"
+import styled from "styled-components"
+import theme from "../../../Assets/Theme"
```

Comment:
#minor it's better to use absolute paths vs. relative. Makes it easier to move or copy/paste this file around if needed.